Source/VMware.vSphereDSC/VMware.vSphereDSC.psm1

```diff
@@ -16,1335 +16,1451 @@ THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
 
 Using module '.\VMware.vSphereDSC.Helper.psm1'
 
-enum Ensure
-{
-    Absent
-    Present
+enum Ensure {
+  Absent
+  Present
 }
 
-enum ServicePolicy
-{
-    Unset
-    On
-    Off
-    Automatic
+enum ServicePolicy {
+  Unset
+  On
+  Off
+  Automatic
 }
 
-enum Period
-{
-   Day = 86400
-   Week = 604800
-   Month = 2629800
-   Year = 31556926
+enum Period {
+  Day = 86400
+  Week = 604800
+  Month = 2629800
+  Year = 31556926
 }
 
-enum LoggingLevel
-{
-    Unset 
-    None
-    Error
-    Warning
-    Info
-    Verbose
-    Trivia
+enum LoggingLevel {
+  Unset
+  None
+  Error
+  Warning
+  Info
+  Verbose
+  Trivia
 }
 
-class BaseDSC
-{
-    <#
+class BaseDSC {
+  <#
     .DESCRIPTION
 
     Name of the Server we are trying to connect to. The Server can be a vCenter or ESXi.
     #>
-    [DscProperty(Key)]
-    [string] $Server
+  [DscProperty(Key)]
+  [string] $Server
 
-    <#
+  <#
     .DESCRIPTION
 
     Credentials needed for connection to the specified Server.
     #>
-    [DscProperty(Mandatory)]
-    [PSCredential] $Credential
+  [DscProperty(Mandatory)]
+  [PSCredential] $Credential
 
-    <#
+  <#
     .DESCRIPTION
 
     Established connection to the specified vSphere Server.
     #>
-    hidden [PSObject] $Connection
+  hidden [PSObject] $Connection
 
-    <#
+  <#
     .DESCRIPTION
 
     Imports the needed VMware Modules.
     #>
-    [void] ImportRequiredModules()
-    {
-	    $savedVerbosePreference = $global:VerbosePreference
-        $global:VerbosePreference = 'SilentlyContinue'
-		
-        Import-Module -Name VMware.VimAutomation.Core
-		
-		$global:VerbosePreference = $savedVerbosePreference
-    }
+  [void] ImportRequiredModules() {
+    $savedVerbosePreference = $global:VerbosePreference
+    $global:VerbosePreference = 'SilentlyContinue'
 
-    <#
+    #        Set-PowerCLIConfiguration -ParticipateInCeip:$false -DisplayDeprecationWarnings:$false -Scope Session -Confirm:$false
```

Comment:
Remove this commented code

Reply:
Line removed.
I appended that change to the commit.